Update two-factor-authentication.md to match new requirement for otp_… (#91)
AdobeDocs/commerce-testing: src/pages/functional-testing-framework/two-factor-authentication.md

bin/magento config:set twofactorauth/general/force_providers google
```

> Now set the OTP window to `2`:

Reviewer: ```suggestion
Now set the OTP window to `29`:
```

> Now set the OTP window to `29` seconds:

Reviewer: ```suggestion
Now set the OTP window to `2`:
```

<InlineAlert variant="info" slots="text" />

> The default OTP window is `1` second in [2.4.7](https://experienceleague.adobe.com/docs/commerce-operations/release/notes/adobe-commerce/2-4-7.html).

Reviewer: ```suggestion
The default OTP window is `1` in [2.4.7](https://experienceleague.adobe.com/docs/commerce-operations/release/notes/adobe-commerce/2-4-7.html). This setting determines the number of past and future codes that remain valid. A value of `1` indicates that the current OTP code plus one code in the past and one code in the future remain valid at any given point in time.  Changing the value to `2` allows the OTP codes to remain valid for longer during automated testing.
```

```bash

> bin/magento config:set twofactorauth/google/otp_window 29

Reviewer: ```suggestion
bin/magento config:set twofactorauth/google/otp_window 2
```

> The default OTP window is `1` in [2.4.7](https://experienceleague.adobe.com/docs/commerce-operations/release/notes/adobe-commerce/2-4-7.html). This setting determines the number of past and future codes that remain valid. A value of `1` indicates that the current OTP code plus one code in the past and one code in the future remain valid at any given point in time.  Changing the value to `2` allows the OTP codes to remain valid for longer during automated testing.

Reviewer: ```suggestion
The default OTP window is `1` in [2.4.7](https://experienceleague.adobe.com/docs/commerce-operations/release/notes/adobe-commerce/2-4-7.html).
```

> bin/magento config:set twofactorauth/google/otp_window 2

Reviewer: ```suggestion
bin/magento config:set twofactorauth/google/otp_window 29
```
